scripts/reboot

```diff
@@ -30,6 +30,36 @@ function clear_warm_boot()
     /sbin/kexec -u || /bin/true
 }
 
+SCRIPT=$0
+
+function show_help_and_exit()
+{
+    echo "Usage ${SCRIPT} [options]"
+    echo "    Request rebooting the device. Invoke platform tool when available."
```

Comment:
Suggest change "platform tool" to "platform-specific tool"

Reply:
Thanks! Done.